jobs/run_initialize.sh

```diff
@@ -209,14 +209,17 @@ while [[ $TIME_COUNT -lt 60 ]] && [[ $PROC_COUNT -gt 0 ]]; do
     if [[ $finished_devstack -eq 0 ]]; then
         ps -p $pid_devstack > /dev/null 2>&1 || finished_devstack=$?
         [[ $finished_devstack -ne 0 ]] && PROC_COUNT=$(( $PROC_COUNT - 1 ))
+        [[ $finished_devstack -ne 0 ]] echo `date -u +%H:%M:%S` "Finished building devstack"
```

Comment:
since you do 
a && b
a && c
and b is just a variable value change, can't you just do:
a && b && c
Any reason why that would not work?